_resources/port1.0/group/php5extension-1.0.tcl

```diff
@@ -216,7 +216,7 @@ proc php5extension.setup {extensions version {source ""}} {
         destroot.target             install-modules install-headers
 
         livecheck.type              none
-        livecheck.url               http://www.php.net/downloads.php
+        livecheck.url               https://www.php.net/downloads.php
```

Comment:
That link does not work for me. Are you sure about it?

Reply:
hrm, checking my shell history and it looks like I tested this yesterday, but checking it again now and it fails. 

Thanks for the catch!

Reply:
ah, I see, messed up on the check. Fixing.